src/services/aws/bankDetails/constants.ts

```diff
@@ -0,0 +1,40 @@
+import { CreateRecipientRequest, WiseRequest } from "types/aws";
+
+export const REQUESTS = {
```

Comment:
it's better to move these requests to redux/query, specially for `GET` requests to take advantage of caching 

with current setup, 
everytime `/banking` is rendered (i.e.  user goes to `/banking`, then `/edit-profile`, then `/banking`), `GET /v1/currencies` is fired

Reply:
Reason was that we're targeting one and the same AWS endpoint, but now that you mention caching, I think it might even be better to write all these calls as separate endpoints in `aws.injectEndpoints` :thinking: This would also solve remove the need for [useTypedWiseMutation](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2439/files#diff-ffd7f4b3dc2dac7b058b975c84453fb7bb7badead212d7afa60b65884f6f4c88R18), as each endpoint would have a defined expected return type.
Will update as per your suggestion :+1:

Reply:
See https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2439/commits/7b0a6f7f8ec805c18446dd5e5ff6353abf27874d